### ISSUE TITLE
[studio-ui] AWS File upload no longer works

### ISSUE THREAD

**Issue by AndyReiter** [craftercms/craftercms#6453]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [x] The issue is in the develop branch
- [X] The issue is in the latest released 4.1.x
- [ ] The issue is in the latest released 4.0.x
- [ ] The issue is in the latest released 3.1.x

### Describe the issue

When trying to upload an asset using the "aws-file-upload" controller in Crafter 4.1.x version, it fails. The following error is found in the console.

Uncaught TypeError: Cannot set properties of null (setting 'disabled')
at HTMLInputElement._onChange (aws-file-upload.js:148:61)
at HTMLInputElement.P (VM2498 utilities.js:11:8669)
_onChange @ aws-file-upload.js:148
P @ VM2498 utilities.js:11
upload.json:1
        
POST http://localhost:8080/studio/api/1/services/api/1/aws/s3/upload.json?&_csrf=38bdd80a-68b8-4737-b501-565c52b8a874 401 (Unauthorized)
aws-file-upload.js:93 Uncaught TypeError: Cannot set properties of null (setting 'disabled')
at Object.upload (aws-file-upload.js:93:65)
at HTMLIFrameElement.F (utilities.js:16:4131)
at HTMLIFrameElement.P (VM2498 utilities.js:11:8669)

It is confirmed that there are no permission-related problems in accessing the s3 bucket from my local install. The same configuration and access configurations are working in Crafter 3.1.x version.

### Steps to reproduce

Steps:

1. Start Crafter version 4.1.x (I tested on 4.1.0 and 4.1.2)
2. Create a test site
3. Add the aws file uploader in configurations
3.1 Edit the 'Content type editor Config'
3.2 Add following config

```
<control>
<name>aws-file-upload</name>
<icon>
<class>fa-upload</class>
<stackedclass>fa-magic</stackedclass>
</icon>
</control>
```

3.3 Save the changes
4. Add the controller to form-definition
4.1 Open article under 'Content Types'
4.2 Search and add the controller 'AWS File Upload'
4.3 Set profile id as 's3-images'
4.4 Save & Close
5. Add the S3 credentials to AWS profiles
5.1 Edit AWS Profiles under config
5.2 Add the following profile with correct credentials.

```
<aws>
<version>2</version>
<s3>
<profile>
<id>s3-images</id>
<credentials>
<accessKey>accessKeyHere</accessKey>
<secretKey>secretKeyHere</secretKey>
</credentials>
<region>eu-west-2</region>
<bucketName>someBucketName/images</bucketName>
</profile>
</s3>
</aws>
```

5.2 Save & Close
6. Upload Image
6.1 Edit an article
6.2 Upload a image
6.3 Check console for errors - Check screenshot

### Relevant log output

```shell
Nothing is reported in the tomcat catalina.out or any crafter logs.
```

### Screenshots and/or videos

![image](https://github.com/craftercms/craftercms/assets/42217570/c2824388-3c54-488c-b86c-6a0ea5added1)

**Comment by sumerjabri** [comment]
AWS File Upload was removed in favor of S3 as a `data-source`; see https://github.com/craftercms/craftercms/issues/2508

This issue will be fixed in the spirit of backward compatibility in the near term and will be disabled again in the future. AWS S3 `data-source` and/or Blob Stores should be used instead.

**Comment by rart** [comment]
@jvega190 pls port https://github.com/craftercms/studio-ui/pull/3645 to develop

**Comment by AndyReiter** [comment]
Confirmed fixed in the latest build of the support/4.1.x build.

**Comment by AndyReiter** [comment]
Confirmed in the latest build of the `develop` branch.
